AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                        f"P1: Shadow filter method set to {filter_method}")
+                    Report.result(test_shadow_filter_method,
+                                  current_filter_method == SHADOW_FILTER_METHOD[filter_method])
+
+                    # 18. Edit the Filtering sample count parameter.
+                    if current_filter_method in (1, 3):
+                        light_component.set_component_property_value(
+                            AtomComponentProperties.light('Filtering sample count'), 64)
+                        Report.result(Tests.edit_filtering_sample_count,
+                                      light_component.get_component_property_value(
+                                          AtomComponentProperties.light('Filtering sample count')) == 64)
+
+                    # 19. Edit the ESM Exponent parameter.
+                    if current_filter_method in (2, 3):
+                        light_component.set_component_property_value(
+                            AtomComponentProperties.light('ESM exponent'), 5000)
+                        Report.result(Tests.edit_esm_exponent,
+                                      light_component.get_component_property_value(
+                                          AtomComponentProperties.light('ESM exponent')) == 5000)
+
+                # 20. Disable Shadows (re-enabled after test for game mode verification):
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shadow'), False)
+                general.idle_wait_frames(1)
+                Report.result(
+                    Tests.disable_shadows,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Enable shadow')) is False)
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shadow'), True)
+
+            # Shutter tests for applicable light types:
+            if current_light_type in (2, 7):
+                # 21. Edit the Inner angle parameter:
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Inner angle'), 0.5)
+                Report.result(Tests.edit_inner_angle,
+                              light_component.get_component_property_value(
+                                  AtomComponentProperties.light('Inner angle')) == 0.5)
+
+                # 22. Edit the Outer angle parameter:
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Outer angle'), 90)
+                Report.result(Tests.edit_outer_angle,
+                              light_component.get_component_property_value(
+                                  AtomComponentProperties.light('Outer angle')) == 90)
+
+            # Disable/Enable Shutters tests:
+            if current_light_type == 2:
```

Comment:
same here use `LIGHT_TYPE` dict reference for the number

Reply:
Changed to:
if light_type == 'spot_disk':